source/tutorials/importing.rst

```diff
@@ -198,73 +297,71 @@ Importing Data
      --output-path paired-end-demux.qza \
      --input-format PairedEndFastqManifestPhred64
 
+Sequences without quality information (i.e. FASTA)
+--------------------------------------------------------
+
 .. _`casava import`:
 
-Casava 1.8 single-end demultiplexed fastq
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Unfortunately, there is currently no way to import data files which contain quality-filtered FASTA sequences associated with sample IDs.
```

Comment:
How about the [`QIIME1DemuxFormat`](https://github.com/qiime2/q2-types/blob/master/q2_types/per_sample_sequences/_format.py#L242-L273)? We have a tutorial that has a brief usage example: https://docs.qiime2.org/2018.8/tutorials/otu-clustering/

I could drop in a brief example of this format here in this section if that helps.

Reply:
Oh, so this format does exist already? Does this mean [the issue (#197) I filed in q2-types](https://github.com/qiime2/q2-types/issues/197) isn't necessary?

If that's the case, then yes having you put a brief example here would be great!

Reply:
No, I think that issue makes sense to me, since we don't currently support importing per-sample demuxed FASTA. The `QIIME1DemuxFormat` is a single-file demuxed format that uses a weird header syntax. It is great for QIIME 1 data, but I haven't really seen too many other tools that use it.

Reply:
ah ok, I see. Yeah, some way to specify these differences might be useful, and to give users at least one option to import FASTA sequences (even if it's a pain to format the sequences in this way...)